www/wordpress/Portfile

```diff
@@ -1,27 +1,29 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem              1.0
+PortSystem          1.0
```

Comment:
This commit did not "upgrade to 5.5.1"; it just changed the whitespace, so that's all the commit message should have said.